applications/luci-app-wechatpush/Makefile

```diff
@@ -1,54 +1,53 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=luci-app-wechatpush
-PKG_VERSION:=3.4.1
+PKG_VERSION:=3.4.2
+PKG_RELEASE:=
```

Comment:
we manually fixed these format issues, now you add them again.
please don't just do copy&paste